packages/components/src/InputDate/InputDate.tsx

```diff
@@ -0,0 +1,57 @@
+import { omit } from "lodash";
+import React from "react";
+import { CommonFormFieldProps, FormField, FormFieldProps } from "../FormField";
+import { DatePicker } from "../DatePicker";
+
+interface InputDateProps
+  extends CommonFormFieldProps,
+    Pick<
+      FormFieldProps,
+      | "readonly"
+      | "disabled"
+      | "onEnter"
+      | "onFocus"
+      | "inputRef"
+      | "validations"
+      | "placeholder"
+      | "onChange"
+      | "onBlur"
+    > {
+  value?: Date;
+  onChange(newValue: Date): void;
+  onClick?(
+    event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+  ): void;
```

Comment:
Do you need this?

Reply:
Nope. Lol.

https://github.com/GetJobber/atlantis/pull/783/commits/ab3816f933e7ec1c5b9c0358facfc963b4222771